ds-prep-python-dictionaries/main.py

```diff
@@ -0,0 +1,26 @@
+person = {
+    "first_name": "Gene",
+    "last_name": "Sun",
+    "hobby": "gaming"
+}
+
+print(person)
+
+first_name = person["first_name"]
+
+last_name = person.get("last_name")
+
+middle_name = person.get("middle_name")
+
+print(first_name, middle_name, last_name)
+#I expect "Gene Sun" as the output.
+#Output was "Gene None Sun" - interesting how it still returned "None" for middle_name.
```

Comment:
That's because, if you jump into the official Python docs for [`dict.get`](https://docs.python.org/3/library/stdtypes.html#dict.get), the `get` method takes an optional second argument to use as the default value if the `key` doesn't exist. If the second argument isn't provided, it defaults to `None`.

A follow-up question for you is, what happens if you try to get the value of a non-existent key with bracket notation (ex: `person["middle_name"]`)?